src/main/java/com/commands/FireRocketCommand.java

```diff
@@ -22,10 +22,15 @@ public FireRocketCommand(String direction) {
 	
 	public void execute(Vehicle vehicle) {
 		// TODO: set Positions
+		if (!vehicle.isRocketReady()) {
+			return;
+		}
 		World world = vehicle.getWorld();		
 		Rocket rocket  = new Rocket (this.direction);
-		rocket.setTargetSpeed(100);
+		//TODO speed of rocket

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)